apps/darts-modernisation/darts-migration/darts-api-migration.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: darts-api-migration
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-api-migration
+  chart:
+    spec:
+      chart: ./stable/darts-api
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
+  values:
+    java:
+      replicas: 1
+      # pin the version of the container image from the commit build here - update string removed
+      image: sdshmctspublic.azurecr.io/darts/api:prod-ab41541-20240130140812
```

Comment:
You ar adding imagePolicy, can you add it to the images?

Reply:
theres a need to pin the image tag to a specific commit hash for the db schema cant use latest